twitchio/ext/eventsub/models.py

```diff
@@ -75,11 +75,18 @@ def __init__(self, request: web.Request):
         self.message_id: str = request.headers["Twitch-Eventsub-Message-Id"]
         self.message_retry: int = int(request.headers["Twitch-Eventsub-Message-Retry"])
         self.message_type: str = request.headers["Twitch-Eventsub-Message-Type"]
-        self.signature: str = request.headers["Twitch-Eventsub-Message-Signature"]
+        try:
+            self.signature: str = request.headers["Twitch-Eventsub-Message-Signature"]
+        except:
+            self.signature: str = ""
```

Comment:
```suggestion
        self.signature: str = request.headers["Twitch-Eventsub-Message-Signature"]
```
signatures are not optional, if not present this should error out